src/core/data/grid/gridlayout.h

```diff
@@ -774,6 +775,12 @@ namespace core
             return localPoint;
         }
 
+        template<typename T>
+        auto AMRToLocal(Point<T, dimension> AMRPoint) const
+        {
+            return AMRToLocal(AMRPoint());
```

Comment:
I see this as a trick... but not the way I would program it in the first place if I didn't know it'd break on PGI.... So I think it will not be clear for future me this has to be done this way.... We should maybe comment why this is done this way (pgi...) so we don't break it later when we'll have forgotten the reason...

Reply:
the issue is constructor resolution, something to try might just be to move the Point(std::array constructor up so it's first) in the list

Reply:
I knew I had this issue before https://github.com/PhilipDeegan/PHARE/blob/gpu/src/core/numerics/interpolator/interpolator.h#L578

Reply:
ok let's try to move the Ctor first then and if it works add a comment so it stays there?

Reply:
resolved via enable_if on the Point parameter pack constructor